jumble/vite.config.ts

```diff
@@ -81,6 +81,7 @@ export default defineConfig({
     },
     headers: {
       "Service-Worker-Allowed": "/data/",
+      "access-control-allow-origin": "*",
```

Comment:
Is this still needed for jumble dev to load the module? If so let's have a comment for why this is here